enterprise/cypher/pom.xml

```diff
@@ -51,248 +48,16 @@
   </profiles>
 
   <build>
-    <plugins>
-
-      <plugin>
-        <groupId>net.alchim31.maven</groupId>
-        <artifactId>scala-maven-plugin</artifactId>
-        <version>3.2.0</version>
-        <executions>
-          <execution>
-            <id>scala-compile</id>
-            <goals>
-              <goal>add-source</goal>
-              <goal>compile</goal>
-            </goals>
-            <phase>process-resources</phase>
-          </execution>
-          <execution>
-            <id>scala-test-compile</id>
-            <phase>${scala-test-compile-phase}</phase>
-            <goals>
-              <goal>testCompile</goal>
-            </goals>
-          </execution>
-        </executions>
-        <configuration>
-          <args>
-              <arg>-Xmax-classfile-name</arg>
-              <arg>100</arg>
-              <!-- arg>-deprecation</arg -->
-          </args>
-          <jvmArgs>
-            <jvmArg>-Xms64m</jvmArg>
-            <jvmArg>-Xmx1024m</jvmArg>
-          </jvmArgs>
-        </configuration>
-      </plugin>
-
-      <plugin>
-        <groupId>org.neo4j.build.plugins</groupId>
-        <artifactId>clirr-maven-plugin</artifactId>
-        <configuration>
-          <includes>
-            <include>org/neo4j/cypher</include>
-          </includes>
-
-          <excludes>
-            <!-- Ignore all Cypher internals -->
-            <exclude>org/neo4j/cypher/internal/*</exclude>
-          </excludes>
-        </configuration>
-      </plugin>
-
-      <plugin>
-          <groupId>org.apache.maven.plugins</groupId>
-          <artifactId>maven-enforcer-plugin</artifactId>
-          <version>1.3.1</version>
-          <executions>
-              <execution>
-                  <id>enforce-dont-exist</id>
-                  <goals>
-                      <goal>enforce</goal>
-                  </goals>
-                  <configuration>
-                      <rules>
-                          <requireFilesDontExist>
-                              <files>
-                                  <file>cypher-compiler-3.2</file>
-                                  <file>cypher-compiler-3.1</file>
-                                  <file>cypher-compiler-3.0</file>
-                                  <file>cypher-compiler-2.3</file>
```

Comment:
Why is this removed? It can happen if you switch branches or fail at forward-merging, and has been known to prevent bad check-ins.

Reply:
It's already defined in the parent POM and is inherited. All cypher modules inherits from  `community/cypher/pom.xml`, even enterprise.